02-s3/README.md

```diff
@@ -121,21 +121,30 @@ Add an object to your bucket:
 
 _How would you copy the contents of the directory to the top level of your bucket?_
 
+> Answer: you can use aws `s3 localdir S3://bucket --recursive`
+
```

Comment:
More specifically: `aws s3 cp --recursive local s3://my-bucket` or `aws s3 sync local s3://my-bucket`

Don't forget to include the command in there (missing `cp` or `sync`)